airflow/auth/managers/base_auth_manager.py

```diff
@@ -24,9 +24,11 @@
 from airflow.utils.log.logging_mixin import LoggingMixin
 
 if TYPE_CHECKING:
+    from flask import Flask
+
     from airflow.auth.managers.models.base_user import BaseUser
     from airflow.cli.cli_config import CLICommand
-    from airflow.www.security import AirflowSecurityManager
+    from airflow.www.security_manager import AirflowSecurityManagerV2
 
 
 class BaseAuthManager(LoggingMixin):
```

Comment:
hmm yeah ok. Though in the current state of things, a vanilla AirflowSecurityManagerV2 won't get you very far because it lacks some useful functions, but the end goal is that all auth providers shouldn't need to write their own security manager.

Reply:
100% agree :)